ruby/db/migrate/20191209212641_create_orders.rb

```diff
@@ -0,0 +1,12 @@
+class CreateOrders < ActiveRecord::Migration[5.1]
+  def up
+    create_table :orders do |t|
+      t.integer :user
```

Comment:
You are creating here field named user which has a type of integer. Is it correct?